lib/fathead/redis_commands/parse.py

```diff
@@ -10,30 +10,43 @@
 f = open(output, "w");
 
 tree = lxml.html.parse("download/raw.dat").getroot()
-commands = tree.find_class("command")
+
+"""
+Extract all <li> elements
+First 8 elements are website's meta links, so ignore them
+"""
+elements = tree.findall(".//li")[8:]
 
 data = {}
 
-for command in commands:
+# Iterate over all desired <li> elements
+for element in elements:
+
+    # Find <a> tag within this <li> element
+    link = element.find('a')
+    command_url = "%s%s" % (url, link.get('href'))  # Save `href` attribute for this <a> tag
```

Comment:
Normally I prefer to use `str.format()` instead of `%`, what about this:

```python
    # Save `href` attribute for this <a> tag
    command_url = "{}{}".format(url, link.get('href'))
```

Please keep in mind that `%` could be deprecated in the future. Please see [PEP 3101](https://www.python.org/dev/peps/pep-3101/) for more information.